frontend/src/components/RotateCredentials.vue

```diff
@@ -238,71 +234,77 @@ export default {
+          successMessage: `Preparing rotation of certificate authorities for ${this.shootName}`,
           heading: 'Do you want to prepare the rotation of certificate authorities?',
-          messages: [
-            'New Certificate Authorities will be created and added to the bundle (together with the old Certificate Authorities).'
+          actions: [
+            'New Certificate Authorities will be created and added to the bundle (old Certificate Authorities will remain in the bundle)'
           ]
         },
         'rotate-ca-complete': {
           caption: this.isLoading
             ? 'Completing certificate authorities rotation'
             : 'Complete Certificate Authorities Rotation',
           errorMessage: 'Could not complete the rotation of certificate authorities',
-          successMessage: `Rotation of certificate authorities completed for ${this.shootName}`,
+          successMessage: `Completing rotation of certificate authorities for ${this.shootName}`,
           heading: 'Do you want to complete the rotation of certificate authorities?',
-          messages: [
-            'Ensure that all parties (end-users, CD pipelines etc.) have updated their kubeconfig for this Shoot cluster since the Certificate Authorities rotation was last initiated. Otherwise the client requests will fail as the old Certificate Authorities will be dropped from the bundle.'
+          actions: [
+            'Old Certificate Authorities will be dropped from the bundle',
+            'Ensure that all parties (end-users, CD pipelines etc.) have updated their kubeconfig for this Shoot cluster since the rotation was prepared. Otherwise the client requests will fail as the old Certificate Authorities will be dropped from the bundle'
           ]
         },
         'rotate-observability-credentials': {
           caption: this.isLoading ? 'Scheduling observability passwords rotation' : 'Start Observability Passwords Rotation',
           errorMessage: 'Could not start the rotation of observability passwords',
           successMessage: `Rotation of observability passwords started for ${this.shootName}`,
           heading: 'Do you want to start the rotation of observability passwords?',
-          messages: [
+          actions: compact([
+            'The current observability passwords will be invalidated',
+            'New observability passwords will be generated',
             this.isAdmin
-              ? 'Note Operator: This will invalidate the user observability passwords. Operator passwords will be rotated automatically. There is no way to trigger the rotation manually.'
-              : 'The current observability passwords will be invalidated.'
-          ]
+              ? 'Note Operator: This will invalidate the user observability passwords. Operator passwords will be rotated automatically. There is no way to trigger the rotation manually'
```

Comment:
btw this will change with https://github.com/gardener/dashboard/issues/1318
cc @istvanballok